contrib/go/src/python/pants/contrib/go/subsystems/go_distribution.py

```diff
@@ -20,7 +20,7 @@ class GoReleaseUrlGenerator(BinaryToolUrlGenerator):
   _DIST_URL_FMT = 'https://storage.googleapis.com/golang/go{version}.{system_id}.tar.gz'
 
   _SYSTEM_ID = {
-    'darwin': 'darwin-amd64',
+    'mac': 'darwin-amd64',
```

Comment:
With a clean build, this change is needed to fetch a Go distribution. I believe this is related to the recent Go distribution fetching changes, which are awesome BTW! This is great so every single Go distribution is available as they are released by the Go team.

Reply:
cc @cosmicexplorer : is this expected?

Reply:
Bump @cosmicexplorer - AFAICT this is needed to build.